packages/ui/components/toast/showToast.tsx

```diff
@@ -11,43 +11,51 @@ type IToast = {
 export const SuccessToast = ({ message, toastVisible }: IToast) => (
   <div
     className={classNames(
-      "data-testid-toast-success bg-brand-500 mb-2 flex h-9 items-center space-x-2 rounded-md p-3 text-sm font-semibold text-white shadow-md rtl:space-x-reverse",
+      "data-testid-toast-success bg-brand-500 mb-2 flex h-auto items-center space-x-2 rounded-md p-3 text-sm font-semibold text-white shadow-md rtl:space-x-reverse md:max-w-sm",
       toastVisible && "animate-fade-in-up"
     )}>
-    <Icon.FiCheck className="h-4 w-4" />
+    <span>
+      <Icon.FiCheck className="h-4 w-4" />
+    </span>
```

Comment:
following the `md:max-w-sm` fix, the message pushes the icon. the icon then loses its width. wrapping it in a `span` to preserve its dimensions